tests/mocks/githubrest.py

```diff
@@ -192,9 +192,9 @@ def get_repos_for_user(
             **kwargs,
         )
 
-    def get_enterprise_audit_logs(self, **kwargs: Any):
+    def get_enterprise_audit_logs(self, *, search_phrase: str, **kwargs: Any):
         url = (
             f"{self.base_url}/enterprises/test-enterprise"
-            f"/audit-log?per_page=100&phrase=action:protected_branch.create"
+            f"/audit-log?per_page=100&phrase={search_phrase}"
         )
         self.add_response(url=url, **kwargs)
```

Comment:
since you can call the audit-log endpoint with no search_phrase, it could be handled here, making your if/else in the test unnecessary


```suggestion
    def get_enterprise_audit_logs(self, *, search_phrase: Optional[str], **kwargs: Any):
        url = f"{self.base_url}/enterprises/test-enterprise/audit-log"
        url += f"?per_page={self.per_page}"
        if search_phrase:
            url += f"&phrase={search_phrase}"
        self.add_response(url=url, **kwargs)

```